File: packages/ios-client/Sources/ios-client/Types/TrackBandwidthLimit.swift
/// Type describing maximal bandwidth that can be used, in kbps. 0 is interpreted as unlimited bandwidth.
public typealias BandwidthLimit = Int

/// Type describing bandwidth limitation of a Track
/// An enum of `BandwidthLimit`
public enum TrackBandwidthLimit {
    case BandwidthLimit(BandwidthLimit)
}

File: packages/android-client/MobileWhepClient/src/main/java/com/mobilewhep/client/Models/TrackBandwidthLimit.kt
package com.mobilewhep.client

/**
 * Type describing bandwidth limitation of a Track.
 * Can be`BandwidthLimit` only
 */
sealed class TrackBandwidthLimit {
  /**
   * Type describing maximal bandwidth that can be used, in kbps. 0 is interpreted as unlimited bandwidth.
   */
  class BandwidthLimit(
    val limit: Int
  ) : TrackBandwidthLimit()
}
